build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Heroku generically uses the `build` script during the build phase. This is my attempt to cleanly run different builds for different heroku environments based on the environment variables set specific to each heroku environment. Different builds are necessary for different environment variables which need to be setup in the build phase for ucp-ui.